### ISSUE TITLE
It's a significant bug in this plugin.

### ISSUE THREAD

**Issue by lhyyydsnbb** [flutternetwork/WiFiFlutter#348]
When using this plugin to connect to a microcontroller hotspot that has no internet connection.after creat socket client connet the microcontroller and write my message to the socket. when this microcontroller hotspot was close,I using this plugin connect normal wifi ,the hold App network is was cannot using.because this plugin and socket was significant bug.Let me goto the microcontroller hotspot was close,if I using other plugin connect normal wifi,the plugin is must Jump to system wifi setting,and I click the normal wifi,the App is was normal, if using your plugin automated connect wifi, bug will happening!

**Comment by Jay-57blocks** [comment]
Hi, has this problem been solved?

**Comment by Jay-57blocks** [comment]
@lhyyydsnbb Hi, has this problem been solved?

**Comment by lhyyydsnbb** [comment]
@Jay-57blocks yes,the The wifi_iot plugin contains a function.' WiFiForIoTPlugin.forceWifiUsage();' I use this function wrapped in my use code. and than successfully executed the code. you can looking this function.' WiFiForIoTPlugin.forceWifiUsage(); in the wifi_iot plugin;
